website/docs/docs/building-a-dbt-project/hooks-operations.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Hooks & Operations"
+id: "hooks-operations"
+---
+
+## Related documentation
+* [pre-hook & post-hook](pre-hook-post-hook)
+* [on-run-start & on-run-end](on-run-start-on-run-end)
+* [`run-operation` command](run-operation)
+
+## Assumed knowledge:
+* [Project configurations](dbt_project.yml.md)
```

Comment:
```suggestion
* [Project configurations](dbt_project.yml)
```

This is actually a broken link! I guess there's a change we should make in the link checker